src/main/kotlin/no/nav/familie/baks/mottak/task/OpprettSøknadBehandlingISakTask.kt

```diff
@@ -0,0 +1,87 @@
+package no.nav.familie.baks.mottak.task
+
+import no.nav.familie.baks.mottak.integrasjoner.BehandlingStatus
+import no.nav.familie.baks.mottak.integrasjoner.BehandlingType
+import no.nav.familie.baks.mottak.integrasjoner.FagsakStatus
+import no.nav.familie.baks.mottak.integrasjoner.JournalpostClient
+import no.nav.familie.baks.mottak.integrasjoner.KontantstøtteOppgaveMapper
+import no.nav.familie.baks.mottak.integrasjoner.KsSakClient
+import no.nav.familie.baks.mottak.integrasjoner.RestMinimalFagsak
+import no.nav.familie.baks.mottak.integrasjoner.finnesÅpenBehandlingIFagsak
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = OpprettSøknadBehandlingISakTask.TASK_STEP_TYPE,
+    beskrivelse = "Oppretter behandling i sak etter automatisk journalføring av søknad",
+)
+class OpprettSøknadBehandlingISakTask(
```

Comment:
Har lyst til å gjenbruke denne til barnetrygd derfor det bare er "Sak".
Vet ikke om noen har noen tanker på dette? Eller om det bør vare en separat opprettelse task for BA?